google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -14,7 +14,7 @@
   ~ limitations under the License.
   -->
 
-<idea-plugin version="2">
+<idea-plugin>
```

Comment:
Is the `version` attribute no longer needed? I remember seeing warnings during Gradle builds from the IDEA plugin saying that because no `version` attribute is specified, something wouldn't happen.

Reply:
hmm. I haven't seen that warning I'll take a look